releases/_posts/2021-06-23-spark-release-3-0-3.md

```diff
@@ -0,0 +1,64 @@
+  - [[SPARK-34922]](https://issues.apache.org/jira/browse/SPARK-34922): Use better CBO cost function
+  - [[SPARK-34963]](https://issues.apache.org/jira/browse/SPARK-34963): Nested column pruning fails to extract case-insensitive struct field from array
+  - [[SPARK-34970]](https://issues.apache.org/jira/browse/SPARK-34970): Redact map-type options in the output of explain()
+  - [[SPARK-35080]](https://issues.apache.org/jira/browse/SPARK-35080): Correlated subqueries with equality predicates can return wrong results
+  - [[SPARK-35096]](https://issues.apache.org/jira/browse/SPARK-35096): foreachBatch throws ArrayIndexOutOfBoundsException if schema is case Insensitive
+  - [[SPARK-35106]](https://issues.apache.org/jira/browse/SPARK-35106): HadoopMapReduceCommitProtocol performs bad rename when dynamic partition overwrite is used
+  - [[SPARK-35227]](https://issues.apache.org/jira/browse/SPARK-35227): Replace Bintray with the new repository service for the spark-packages resolver in SparkSubmit
+  - [[SPARK-35296]](https://issues.apache.org/jira/browse/SPARK-35296): Dataset.observe fails with an assertion
+  - [[SPARK-35482]](https://issues.apache.org/jira/browse/SPARK-35482): case sensitive block manager port key should be used in BasicExecutorFeatureStep
+  - [[SPARK-35493]](https://issues.apache.org/jira/browse/SPARK-35493): spark.blockManager.port does not work for driver pod
+  - [[SPARK-35659]](https://issues.apache.org/jira/browse/SPARK-35659): Avoid write null to StateStore
+  - [[SPARK-35673]](https://issues.apache.org/jira/browse/SPARK-35673): Spark fails on unrecognized hint in subquery
+  - [[SPARK-35679]](https://issues.apache.org/jira/browse/SPARK-35679): Overflow on converting valid Timestamp to Microseconds
+  - [[SPARK-34697]](https://issues.apache.org/jira/browse/SPARK-34697): Allow DESCRIBE FUNCTION and SHOW FUNCTIONS explain about || (string concatenation operator)
+  - [[SPARK-34772]](https://issues.apache.org/jira/browse/SPARK-34772): RebaseDateTime loadRebaseRecords should use Spark classloader instead of context
+  - [[SPARK-35127]](https://issues.apache.org/jira/browse/SPARK-35127): When we switch between different stage-detail pages, the entry item in the newly-opened page may be blank
+  - [[SPARK-35168]](https://issues.apache.org/jira/browse/SPARK-35168): mapred.reduce.tasks should be shuffle.partitions not adaptive.coalescePartitions.initialPartitionNum
+  - [[SPARK-35566]](https://issues.apache.org/jira/browse/SPARK-35566): Fix number of output rows for StateStoreRestoreExec
+  - [[SPARK-35714]](https://issues.apache.org/jira/browse/SPARK-35714): Bug fix for deadlock during the executor shutdown
+
+### Dependency Changes
+
+While being a maintence release we did still upgrade some dependencies in this release they are:
+  - [[SPARK-35210]](https://issues.apache.org/jira/browse/SPARK-35210): Upgrade Jetty to 9.4.40 to fix ERR_CONNECTION_RESET issue
+
+### Known issues
```

Comment:
I think known issues mean bugs that are still present? SPARK-34534 says it's fixed in 3.0.3 cc @dongjoon-hyun

Reply:
Ya,  it's fixed.

Reply:
Thanks, I have updated this part: remove these 2 issues to notable changes and added a new known issue: [SPARK-34529](https://issues.apache.org/jira/browse/SPARK-34529)